lib/LambdaHelper.js

```diff
@@ -152,4 +232,57 @@ module.exports = function (aws, queueInitializedPromise, executionStore){
 
         return proxyStore[functionIdentifier];
     };
+
+    this.getCloudedFunctionFromModule = function(modulePath, handlerName, deps, configs) {
```

Comment:
If deps is undefined or null, assign it to an empty array, it'll fail later otherwise...
If functionConfigs is null or undefined, you should assign it to ``` {name: handlerName} ```otherwhise it'll fail later